<launch>

    <env name="ROS_IP" value="127.0.0.1"/>
    <env name="ROS_HOSTNAME" value="$(env ROS_IP)"/>
    
    <param name="ROS_IP" type="str" value="$(env ROS_IP)" />
    <param name="ROS_TCP_PORT" type="int" value="10000" />
    <param name="TCP_NODE_NAME" type="str" value="TCPServer" />
    <param name="TCP_BUFFER_SIZE" type="int" value="1024" />
    <param name="TCP_CONNECTIONS" type="int" value="10" />

    <param name="UNITY_IP" type="str" value="127.0.0.1" />
    <param name="UNITY_SERVER_PORT" type="int" value="5005" />
    <node name="server_endpoint"   pkg="hackev" type="server_endpoint.py" />
</launch>
